modules/core/src/main/java/org/apache/ignite/internal/util/IgniteUtils.java

```diff
@@ -829,6 +831,23 @@ public static <T> CompletableFuture<T> inBusyLockAsync(IgniteSpinBusyLock busyLo
         }
     }
 
+    /**
+     * Method that runs the provided {@code fn} in {@code busyLock} if {@link IgniteSpinBusyLock#enterBusy()} succeed.
```

Comment:
```suggestion
     * Method that runs the provided {@code fn} in {@code busyLock} if {@link IgniteSpinBusyLock#enterBusy()} succeeds. Otherwise it just silently returns.
```

Reply:
Fix it